concrete/blocks/top_navigation_bar/controller.php

```diff
@@ -101,6 +101,7 @@ class Controller extends BlockController implements UsesFeatureInterface, FileTr
     protected $btCacheBlockOutputForRegisteredUsers = true;
     protected $btCacheBlockOutputLifetime = 300;
     protected $btExportFileColumns = ['brandingLogo', 'brandingTransparentLogo'];
+    protected $home;
```

Comment:
I recommend using `$homePage` for the property name though it was `$home` in the original code.